Extension/src/githubAPI.ts

```diff
@@ -182,8 +182,8 @@ export async function getTargetBuildInfo(updateChannel: string): Promise<BuildIn
  * @param updateChannel The user's updateChannel setting.
  * @return The Build if the user should update to it, otherwise undefined.
  */
-export function getTargetBuild(builds: Build[], userVersion: PackageVersion, updateChannel: string): Build | undefined {
-    if (!vscode.workspace.getConfiguration("extensions", null).get<boolean>("autoUpdate")) {
+export function getTargetBuild(builds: Build[], userVersion: PackageVersion, updateChannel: string, isFromSettingsChange: boolean): Build | undefined {
```

Comment:
If you're adding a new parameter to this function, please add it to the function comments as well.

Reply:
Okay, I didn't see those comments.

Reply:
It looks like there are more function comments that should be updated too besides this one.